usmqe_tests/rpm/packagelist.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf8 -*-
+
+tendrl_packages = [
+    "tendrl-alerting",
+    "tendrl-api",
+    "tendrl-api-httpd",
+    "tendrl-ceph-integration",
+    "tendrl-commons",
+    "tendrl-dashboard",
+    "tendrl-gluster-integration",
+    "tendrl-node-agent",
+    "tendrl-node-monitoring",
+    "tendrl-performance-monitoring",
+    ]
```

Comment:
That said, we could fetch all packages from tendrl repository in the same way it's done for tendrl deps and add new test to check that listed tendrl packages are still there. Would that be worth it?